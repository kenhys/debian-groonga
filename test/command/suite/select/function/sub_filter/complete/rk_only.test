table_create Readings TABLE_PAT_KEY ShortText --normalizer NormalizerAuto

table_create Items TABLE_HASH_KEY ShortText
column_create Items readings COLUMN_VECTOR Readings

column_create Readings items_index COLUMN_INDEX Items readings

load --table Items
[
{"_key": "Groonga", "readings": ["グルンガ"]}
]

select Items \
  --filter 'sub_filter(readings, "prefix_rk_search(_key, \\"guru\\")")'
