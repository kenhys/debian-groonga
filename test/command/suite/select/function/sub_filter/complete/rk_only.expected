table_create Readings TABLE_PAT_KEY ShortText --normalizer NormalizerAuto
[[0,0.0,0.0],true]
table_create Items TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Items readings COLUMN_VECTOR Readings
[[0,0.0,0.0],true]
column_create Readings items_index COLUMN_INDEX Items readings
[[0,0.0,0.0],true]
load --table Items
[
{"_key": "Groonga", "readings": ["グルンガ"]}
]
[[0,0.0,0.0],1]
select Items   --filter 'sub_filter(readings, "prefix_rk_search(_key, \\"guru\\")")'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "readings",
          "Readings"
        ]
      ],
      [
        1,
        "Groonga",
        [
          "グルンガ"
        ]
      ]
    ]
  ]
]
